### PR TITLE
Remove weekly decay due to wild mu variance, other elo fixes

### PR DESCRIPTION
**Pull request by kais58**
Following discussion with Stormtrout, weekly decay causing wild swings in player elos, massively inflating them.

### CODE REVIEW

File: lib/tasks/generate_elo_csv.rake [hunk 1]
```diff
@@ -143,8 +127,8 @@ task generate_elo_csv: :environment do
 
     ts_min, ts_max = get_min_max_mu(@players_by_name.values)
 
-    CSV.open("elo_output_decay.csv", "w") do |csv|
-      csv << ["name", "games played", "wins", "losses", "last_played", "ts_elo", "ts_mu", "ts_sigma"]
+    CSV.open("elo_output.csv", "w") do |csv|
+      csv << ["name", "games played", "wins", "losses", "last_played", "ts_elo", "ts_mu", "ts_sigma", "mu_normal"]
```
Comment (mma127) [inline lib/tasks/generate_elo_csv.rake]: You aren't filling in a value for `mu_normal`. What is it for?

File: lib/tasks/generate_elo_csv.rake [hunk 2]
```diff
@@ -53,29 +53,13 @@ task generate_elo_csv: :environment do
     [row["Game-ID"], parse_date(row["Date"]), row["Winner"], match_player]
   end
 
-  def update_sigma_decay(ts_rating, weeks_since_last_played)
```
Comment (mma127) [inline lib/tasks/generate_elo_csv.rake:56]: Why is this necessary?

File: lib/tasks/generate_elo_csv.rake (hunk 1 above)
Comment (kais58) [reply lib/tasks/generate_elo_csv.rake]: my mistake, mixed up files

File: lib/tasks/generate_elo_csv.rake (hunk 2 above)
Comment (kais58) [reply lib/tasks/generate_elo_csv.rake:56]: Short answer is it messes up the calculation for subsequent elo's and has given wild swings up and down to peoples elo's, better off asking Stormtrout for details

File: app/javascript/features/lobby/display/BattleCard.jsx [hunk 3]
```diff
@@ -136,20 +137,20 @@ export const BattleCard = ({ id, rulesetId }) => {
...
+      return `Axis Stomp`
     else if (balance < (0 - min))
-      return "Allied Favoured"
+      return `Allied Favoured`
     else if (balance > min)
-      return "Axis Favoured"
+      return `Axis Favoured`
     else if (balance < 10)
-      return "Perfect"
+      return `Perfect`
```
Comment (mma127) [inline app/javascript/features/lobby/display/BattleCard.jsx:153]: Do we want to show the number?

File: app/services/battle_report_service.rb [hunk 4]
```diff
@@ -1,6 +1,7 @@
 class BattleReportService < ApplicationService
   class BattleReportValidationError < StandardError; end
   MINIMUM_ELAPSED = 300.freeze
+  MINIMUM_RANKED_ELAPSED = 1800.freeze
```
Comment (mma127) [inline app/services/battle_report_service.rb:4]: Stomps under 30 min should count no?

**Review by mma127 — COMMENTED** [review]
What is the release plan for this? Destroy all player rating records and reseed elos?

File: app/services/battle_report_service.rb (hunk 4 above)
Comment (kais58) [reply app/services/battle_report_service.rb:4]: It's basically impossible for a game to be that short without a drop is my understanding, and that probably shouldn't count for ranking
